src/Raven.Client/Documents/Operations/CompareExchange/CompareExchangeValueResultParser.cs

```diff
@@ -59,8 +61,10 @@ public static CompareExchangeValue<T> GetValue(BlittableJsonReaderObject respons
             if (response == null)
                 return null;
 
-            var value = GetValues(response, conventions).FirstOrDefault();
+            var compareExchangeValues = GetValues(response, conventions);
+            var value = compareExchangeValues.FirstOrDefault();
```

Comment:
Looks like debugging code and unrelated to the issue?

Reply:
yes, it is unrelated indeed.